src/views/LostItems/LostItemsListContainer/components/LostItemsList/components/ActualCostDetailsModal/ActualCostDetailsModal.js

```diff
@@ -22,7 +22,7 @@ import {
   ACTUAL_COST_DETAILS_MODAL_DEFAULT,
   LOST_ITEM_STATUSES,
 } from '../../../../../constants';
-import { useLocalizeCurrency } from '../../../../../../../hooks/useLocalizedCurrency/useLocalizeCurrency';
+import useLocalizeCurrency from '../../../../../../../hooks/useLocalizedCurrency/useLocalizeCurrency';
```

Comment:
after updating` src/hooks/index.js` file on the lines of other hooks exports (as suggested below in one of the comments), adjust this import